crates/rethnet_evm_napi/package.json

```diff
@@ -1,25 +1,39 @@
 {
-    "package": "rethnet-evm",
-    "name": "rethnet-evm",
+    "name": "@ignored/edr",
     "version": "0.1.0-dev",
-    "main": "rethnet-evm.node",
+    "main": "index.js",
     "types": "index.d.ts",
-    "license": "MIT",
     "napi": {
-        "name": "rethnet-evm"
-    },
-    "scripts": {
-        "build": "napi build --release",
-        "build:debug": "napi build",
-        "build:tracing": "napi build --release --features tracing",
-        "test": "yarn tsc && mocha --recursive \"test/**/*.ts\" --exit",
-        "clean": "rm -rf rethnet-evm.node"
+        "name": "edr",
+        "triples": {
+            "additional": [
+                "aarch64-apple-darwin",
+                "aarch64-unknown-linux-musl",
+                "x86_64-unknown-linux-musl",
+                "i686-pc-windows-msvc"
+            ]
+        }
     },
+    "license": "MIT",
     "devDependencies": {
-        "@napi-rs/cli": "^2.11.4",
+        "@napi-rs/cli": "^2.16.3",
         "chai": "^4.3.6",
         "@nomicfoundation/ethereumjs-util": "^8.0.0",
         "mocha": "^10.0.0",
         "typescript": "~4.5.2"
+    },
+    "engines": {
+        "node": ">= 12"
```

Comment:
```suggestion
        "node": ">= 18"
```

Reply:
Done in [508bd82](https://github.com/NomicFoundation/hardhat/pull/4415/commits/508bd820d3d579ca155328d25d24142b7238644a)